cli/central-client/src/main/java/org/ballerinalang/central/client/Utils.java

```diff
@@ -480,7 +529,7 @@ public void writeTo(BufferedSink sink) throws IOException {
                 byteConverter = 1024 * 1024;
                 unitName = " MB";
             }
-    
+
             ProgressBar progressBar = new ProgressBar(task, contentLength(), 1000, out,
```

Comment:
Let's move 1000 into a const as well

Reply:
Fix in https://github.com/ballerina-platform/ballerina-lang/pull/41490/commits/99341c281ba6f7557ae22d012736bb1c6f1ce33f